wazuh/wazuh-dashboard-plugins
# Fix design implementation

### Description
-  In the About page delete the API table and leave a React component the same as the previous Angular template.
-  Delete unused components related to the About page.
- In the UpdatesNotification component only ask for available updates if the user doesn't check "Disable update notifications".
- In API Table handle check updates button errors and show a toast.
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard/issues/96

### Evidence

#### About page
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/f63ef799-d74f-46b8-87c5-f9d5281c6e12)

#### Bottom bar notification
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/5904fc9f-595c-4984-b898-c8ae8f555d6e)

#### API Table
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/56f427c2-4d42-4caf-bdc1-c0ef0f56b9c2)

#### Updates status error message
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/1fcfcd95-b4d5-460e-990c-799d1a11d294)

#### Check updates button error
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/b6817794-30b7-49a0-9556-5f589ebde03e)

### Test

- Go to the About page and compare it to the current About page. They must be the same.
- When the bottom bar notification appears, check `Disable update notifications` and click the `Dismiss` button. Reload the page and check the network requests. A request for user preferences (ending in `/me`) should appear, but there should not be a request for available updates (ending in `/updates?checkAvailableUpdates=false`).
- Go to Settings -> API configuration. Disable the network connection and click on `Check Updates` button. A toast should appear with an error.

### Check List
